sections/main-product.liquid

```diff
@@ -559,7 +559,7 @@
     }
   }
 
-  customElements.define('product-modal', ProductModal);
+  if (!customElements.get('product-modal')) customElements.define('product-modal', ProductModal);
 {% endjavascript %}
 
```

Comment:
This is happening when I open the product modal on the product page

https://screenshot.click/17-01-twp0e-mylyk.mp4